### DISCUSSION TITLE
How to enable a module in local?

### DISCUSSION THREAD

**Discussion by AstronoGeek** [AllskyTeam/allsky-modules#199] [Q&A] [answered]
If I go on the web interface (allsky.local), I can't enable a module in the module manager. I can check the checkbox, I then click the green save icon but then, whatever I do, if I go back to this page, it does not leave the module checked. Am I missing something?
![image](https://github.com/user-attachments/assets/dd380b9d-a816-46fd-80a0-43463e9bdae6) 
I am running 2024.12.06

**Comment by Alex-developer** [comment, marked as answer]
> If I go on the web interface (allsky.local), I can't enable a module in the module manager. I can check the checkbox, I then click the green save icon but then, whatever I do, if I go back to this page, it does not leave the module checked. Am I missing something?
> 
> ![image](https://github.com/user-attachments/assets/dd380b9d-a816-46fd-80a0-43463e9bdae6) 
> 
> I am running 2024.12.06
> 
> 

The modules can only be enabled when they are in the right hand column

Alex

**Comment by Alex-developer** [comment]
I'll create a ticket to get them disabled when not activate as it is a little confusing
